packages/theme-ui/README.md

```diff
@@ -160,5 +160,6 @@ export default props => (
 - [Theme Spec](https://theme-ui.com/theme-spec)
 - [Gatsby Plugin](https://theme-ui.com/gatsby-plugin)
 - [API](https://theme-ui.com/api)
+- [Community Discord](https://discord.gg/wAD6p8q)
```

Comment:
Is there not an option for a pretty URL with discord? Also I can’t seem to open this link in the browser on my phone...

Reply:
I don't believe that discord supports phone browsers. When I open it on my phone, it deeplinks to the app when I click the accept button. 

As far as pretty URLs go, I think you need to either earn that with server boosts (people join your server and basically donate with an in app feature) *or* you have to get the server verified like what the Gatsby Discord did https://discordapp.com/verification

The boost option isn't as reliable because users have to keep them active and it takes 30 boosts for the url so verification would make the most sense I think if we wanted this.

Reply:
Thanks for the info! Sounds pretty limiting, but I guess this thing is really intended for gaming not OSS communities...

Reply:
why not slack? just asking don't have an opinion, just curios 😉

Reply:
Eh It's not as big of an issue as it seems I'd say. I've seen and helped plenty of successful open source discords.

You can see that a lot of projects have grown pretty well here https://discordapp.com/open-source.

As for slack, it's a lot easier to not have to make accounts among other reasons. You have a bit more control over categorization and you don't have any message history limits to deal with for free servers are other things.